source/features/pinned-issues-update-time.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import clockIcon from 'octicon/clock.svg';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getRepoGQL, getRepoURL} from '../libs/utils';
+
+interface IssueInfo {
```

Comment:
What is this doing? (Sorry for the random question)

Reply:
This is a TypeScript type. It describes this object:

```js
{
	updatedAt: '........'
}
```

Which is what the API returns, in an object. That other object looks like:

```js
{
	_2345: {
		updatedAt: '........'
	},
	_3455: {
		updatedAt: '........'
	}
}
```

So its type is `Record<string, IssueInfo>`

`Record` is a quick way to define objects with "any" amount of properties. `string` in that case refers to the key of that property and `IssueInfo` refers to the value.

`AnyObject`, used in some places, is short for `Record<string, any>`

Reply:
Thanks